openpype/pipeline/colorspace.py

```diff
@@ -13,12 +13,17 @@
     Logger
 )
 from openpype.pipeline import Anatomy
+from openpype.lib.transcoding import VIDEO_EXTENSIONS, IMAGE_EXTENSIONS
+
 
 log = Logger.get_logger(__name__)
 
 
 class CashedData:
```

Comment:
Pretty sure this is not cashed data. 💰 💵 💲 

```suggestion
class CachedData:
```

However, will need to refactor usages of it too.